src/form/field-handlers/AssetVersionSelector.tsx

```diff
@@ -84,7 +85,7 @@ type UriType = {
     name: string;
     key: string;
     ref: string;
-    icon?: IconValue;
+    icons?: IconValue[];
```

Comment:
Would it make sense to keep `icon` for backwards compatibility?

Reply:
Yea good point for props, let me take another look 👀

Reply:
Ah yea we're good actually, this type is only used internally here, we still support icon in props.